packages/code-du-travail-frontend/pages/contribution/[slug].js

```diff
@@ -37,6 +37,37 @@ class PageContribution extends React.Component {
     return { data };
   }
 
+  function;
+
+  buildTitleAndDescription(breadcrumbs, conventionAnswer, title, description) {
+    if (
+      breadcrumbs &&
+      breadcrumbs.length > 0 &&
+      conventionAnswer &&
+      conventionAnswer.shortName
```

Comment:
there is always a short name, not sure we need the test

Reply:
Le test est positif (beaucoup plus de clics reçus globalement sur ces pages).
A revoir dans 2-3 mois pour confirmer l'impact sur certaines pages.
En savoir plus : https://docs.google.com/document/d/1BjlHoCI8EyML_lyw-lr0WK7RgCgNvcPJKigS54pIQEc/edit?usp=sharing